ruby_event_store/lib/ruby_event_store/specification_reader.rb

```diff
@@ -23,6 +23,25 @@ def each(specification_result)
       end
     end
 
+    # @api private
+    # @private
+    def map(specification_result, &block)
+      result = []
```

Comment:
@mpraglowski style opinion - could use `each_with_object` instead of an extra variable?